content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: blog
+title: "My first-ever KubeCon - KubeCon+CloudNativeCon North America 2023"
+date: 2023-12-05
+slug: My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023
+---
+
+**Author -** Rajdeep Pal, Student, Techno India University
+
+_This post is a guide for beginners to look at huge and overwhelming events like KubeCon._
+
+# Introduction
+
+![KubeCon](https://raw.githubusercontent.com/kubernetes/website/main/content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/pic1.jpg)
+
+This blog is kind of a personal experience cum guide for beginners when they first encounter such a huge and important conference. Although KubeCon is the first conference I attended, it is certainly the largest. Earlier I have been to community-driven conferences such as Google Cloud Community Days Kolkata 2023, Kubernetes Community Days Mumbai 2023, and KubeDay India 2023, they were quite big too but they lasted for a day or two. KubeCon was much larger and more overwhelming for new folks in the field. I attended my first KubeCon virtually as the Dan Kohn Scholarship approved me for virtual attendance. Before deep-diving into the event and my experiences, let's start with the basics of the event or any event hosted by the Cloud Native Computing Foundation.
```

Comment:
Please wrap the lines . Make sure one single line doesn't have more than 10 words or 90 characters.
Follow [Styleguide](https://kubernetes.io/docs/contribute/style/style-guide/#markdown-elements) for more information

Reply:
more reviews to come after once the community approves of this blog's need.